rocky/views/upload_csv.py

```diff
@@ -91,21 +101,18 @@ def proccess_csv(self, form):
         csv_data = io.StringIO(csv_file.read().decode("UTF-8"))
         rows_with_error = []
         try:
-            for rownumber, row in enumerate(csv.reader(csv_data, delimiter=",", quotechar='"')):
-                rownumber += 1  # start at 1 and not 0
-                if len(row) == 1:
-                    object_type_value = row[0]
+            for rownumber, row in enumerate(csv.DictReader(csv_data, delimiter=",", quotechar='"')):
+                if len(row) > 0:  # skip empty lines
```

Comment:
For sequences, (strings, lists, tuples), use the fact that empty sequences are false (https://pep8.org/#programming-recommendations)

```suggestion
                if not row:  # skip empty lines
```

Reply:
That would have done the inverse of what I was hoping for. Commited with a continue statement to keep the complexity low.

Reply:
Sorry, I meant `if row`